db/manager.go

```diff
@@ -120,26 +113,30 @@ func (m *Manager) NewDB(ctx context.Context, creds thread.Credentials, collectio
 // NewDBFromAddr creates a new db from address and prefixes its datastore with base key.
 // Unlike NewDB, this method takes a list of collections added to the original db that
 // should also be added to this host.
-func (m *Manager) NewDBFromAddr(ctx context.Context, creds thread.Credentials, addr ma.Multiaddr, key thread.Key, collections ...CollectionConfig) (*DB, error) {
-	id := creds.ThreadID()
+func (m *Manager) NewDBFromAddr(ctx context.Context, addr ma.Multiaddr, key thread.Key, opts ...NewManagedDBOption) (*DB, error) {
```

Comment:
Nice, this is much cleaner without having to deal with credentials and the thread id (that sent me for a loop when integrating credentials into folder sync)